### ISSUE TITLE
Support managing remarks on back matter resources

### ISSUE THREAD

**Issue by kylelaker** [EasyDynamics/oscal-react-library#689]
### Description

As a catalog author, I want to manage remarks associated with back matter resources.

### Acceptance Criteria

- The `remarks` (`markup-multiline`) for a back matter resource can be created, deleted, or edited
- Modifying the `remarks` field results in a request to REST API, using the most appropriate method/route on the API

### Additional Notes

Keep in mind that `{ "remarks": "" }` is not the same as `{ "remarks": undefined }` (to speak JavaScript here). Deleting the `remarks` and changing `remarks` to be an empty string are not identical operations.

### Implementation

- [ ] I can (or plan to) submit a pull request to implement this
- [ ] Implementing this may result in a breaking change

**Comment by brian-easyd** [comment]
Need an issue to support viewing back-matter remarks (or viewing remarks in general).

**Comment by Bronstrom** [comment]
9/21/23 Sprint Planning - Moved from Sprint 55 to Sprint 56

**Comment by Bronstrom** [comment]
10/05/23 Sprint Planning - Moved from Sprint 56 to Sprint 57

**Comment by brian-easyd** [comment]
Being address as part of EasyDynamics/oscal-prototyping#19

**Comment by brian-easyd** [comment]
11/9/2023 - merging of code on hold pending finalizing of the repo refactoring. Moving to sprint 59.

**Comment by brian-easyd** [comment]
11/29/2023 - clarification: this is complete for enterprise initiatives, but has not been applied to catalogs yet. Will close after repo merge and code can be verifiied in context of catalogs.
Blocked by https://github.com/EasyDynamics/comply0-private/issues/11

**Comment by brian-comply0** [comment]
12/14/2023 - not started yet. moving to sprint 61

**Comment by Bronstrom** [comment]
1/4/2023 - Still blocked. moving to sprint 62

**Comment by brian-comply0** [comment]
2024-01-25 moving to next sprint

**Comment by brian-comply0** [comment]
2024-02-07 moving to next sprint
